stripe/src/main/java/com/stripe/android/view/StripeEditText.kt

```diff
@@ -30,8 +31,15 @@ open class StripeEditText @JvmOverloads constructor(
 
     private var afterTextChangedListener: AfterTextChangedListener? = null
     private var deleteEmptyListener: DeleteEmptyListener? = null
-    var cachedColorStateList: ColorStateList? = null
-        private set
+
+    var defaultColorState: ColorStateList
+        @VisibleForTesting
+        internal set
+    private var externalColorState: ColorStateList? = null
+    @ColorInt
+    private var defaultErrorColor: Int = 0
+    @ColorInt
+    private var externalErrorColor: Int? = null
```

Comment:
I made this change because it kept consistency with how the default color works.  There is a default color set by our style, and if external is set we use that.